src/main/java/org/craftercms/studio/impl/v1/service/configuration/ServicesConfigImpl.java

```diff
@@ -368,6 +369,16 @@ protected SiteConfigTO loadConfiguration(String site) {
              loadFacetConfiguration(configNode, siteConfig);
 
              siteConfig.setPluginFolderPattern(configNode.valueOf(SITE_CONFIG_ELEMENT_PLUGIN_FOLDER_PATTERN));
+
+             String requirePeerReviewValue =
+                     configNode.valueOf(SITE_CONFIG_XML_ELEMENT_WORKFLOW + "/" + SITE_CONFIG_XML_ELEMENT_PUBLISHER +
```

Comment:
Add this field to the sample file as commented out and document what it does in the sample.
@alhambrav please document this new feature in 4. Ping Dejan for help.